docs/system/update.md

```diff
@@ -0,0 +1,146 @@
+---
+title: System Update
```

Comment:
Can you remove parts that are migrated from mx4-wiki (i know most of this is from README file in mobility-platform)
Would be good to be able to shutdown that repo. Or add a mx4 folder and migrate the file mx4 specific there, mx-v specific, hmx specific and C61 specific (it no longer need firmware_update flag.

Reply:
It should also be "System Update".